utility/servicer/module.go

```diff
@@ -466,7 +476,9 @@ func (s *servicer) executeHTTPRelay(serviceConfig *configs.ServiceConfig, payloa
 	}
 
 	// INCOMPLETE(#837): Optimize usage of HTTP client, e.g. connection reuse, depending on the volume of relays a servicer is expected to handle
-	resp, err := (&http.Client{Timeout: time.Duration(serviceConfig.TimeoutMsec) * time.Millisecond}).Do(req)
+	// ADDPR: allow configuration of TLS Settings for HTTPS services
```

Comment:
Do you intend to do it in this PR?

Reply:
Thanks, updated to an `INCOMPLETE` item with issue #887